shared/p2p/hobbits/gossip_handler.go

```diff
@@ -0,0 +1,41 @@
+package hobbits
+
+import (
+	"context"
+	"log"
+
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/p2p"
+)
+
+func (h *HobbitsNode) gossipBlock(message HobbitsMessage, header GossipHeader) {
+	block := &pb.BeaconBlockAnnounce{
+		Hash: header.Hash[:],
+	}
+
+	h.Feed(block).Send(p2p.Message{
+		Ctx: context.Background(),
```

Comment:
please pass in the context as an argument as a argument, do not use `context.Background()` here